.luacheckrc

```diff
@@ -1,6 +1,6 @@
 globals = {
     "_clean_buffer_count",
     "vim",
-    "lean_nvim_ft_options",
+    "lean_nvim_default_filetype",
```

Comment:
would really like to document the behaviour of this global, but i think it would be good to do that in `doc/lean.text` and im not quite sure how to generate the help with lemmy-help (in particular what files are used for it)
It would probably be best to just merge this PR to fix the bug, and then I'll send another PR for a help overhaul